mu-plugins/blocks/query-filter/src/view.js

```diff
@@ -23,6 +47,8 @@ function closeDropdown( store ) {
 	const count = context.wporg.queryFilter.form?.querySelectorAll( 'input:checked' ).length;
 	updateButtons( store, count );
 	document.documentElement.classList.remove( 'is-query-filter-open' );
+
+	toggleOverflowX();
```

Comment:
I think we can remove this and just call the function outside of the `if` conditions on `toggle`

Reply:
I think we might need to keep this as it is, otherwise the [closeDropdown](https://github.com/WordPress/wporg-mu-plugins/blob/262f0816fc6a4b7f271afcf3dd0e66fe2743447c/mu-plugins/blocks/query-filter/src/view.js#L94) within `handleKeydown` won't work, resulting in the filter list being unscrollable after pressing Esc.

Reply:
Ah yes I see what you mean